src/android/InAppBrowser.java

```diff
@@ -579,11 +579,13 @@ public void run() {
                 Resources activityRes = cordova.getActivity().getResources();
                 int backResId = activityRes.getIdentifier("ic_action_previous_item", "drawable", cordova.getActivity().getPackageName());
                 Drawable backIcon = activityRes.getDrawable(backResId);
-                back.setBackground(null);
+                if (Build.VERSION.SDK_INT >= 16)
+                    back.setBackground(null);
                 back.setImageDrawable(backIcon);
                 back.setScaleType(ImageView.ScaleType.FIT_CENTER);
                 back.setPadding(0, this.dpToPixels(10), 0, this.dpToPixels(10));
-                back.getAdjustViewBounds();
+                if (Build.VERSION.SDK_INT >= 16)
+                    back.getAdjustViewBounds();
```

Comment:
Out of curiosity, what purpose do the calls to getAdjustViewBounds serve?  When I read the documentation, it doesn't appear that calling getAdjustViewBounds has any side effects.

https://developer.android.com/reference/android/widget/ImageView.html#getAdjustViewBounds()